Article/Article.js

```diff
@@ -1,29 +1,27 @@
 // Because classes are not hoisted you will need to start your code at the bottom of the page.  Look for the comment "START HERE"
 
 class Article {
-  constructor(domElement) {
+  constructor(article) {
     // assign this.domElement to the passed in domElement
-    this.domElement;
+    this.article = article;
     // create a reference to the ".expandButton" class. 
-    this.expandButton;
+    this.expandButton = this.article.querySelector('.menu-button'); 
     // Using your expandButton reference, update the text on your expandButton to say "expand"
-    
+    this.expandButton.classList.text = 'expand';
     // Set a click handler on the expandButton reference, calling the expandArticle method.
-    
+    this.panelButtons.addEventListener('hover', () => this.togglePanel());
   }
 
   expandArticle() {
     // Using our reference to the domElement, toggle a class to expand or hide the article.
-
+this.expandButton.classList.toggle
+this.expandButton.classList.toggle('expandButton')
   }
 }
 
-/* START HERE: 
-
-- Select all classes named ".article" and assign that value to the articles variable.  
-
-- With your selection in place, now chain .forEach() on to the articles variable to iterate over the articles NodeList and create a new instance of Article by passing in each article as a parameter to the Article class.
+const articles = document.querySelector('.article');
```

Comment:
here we want `querySelectorAll` so we grab every single div with an article class on it